internal/pkg/config/v1/api_key_pair.go

```diff
@@ -1,7 +1,28 @@
 package v1
 
+import (
+	"runtime"
+	"strings"
+
+	"github.com/confluentinc/cli/internal/pkg/secret"
+)
+
 // APIKeyPair holds an API Key and Secret.
 type APIKeyPair struct {
 	Key    string `json:"api_key"`
 	Secret string `json:"api_secret"`
+	Salt   []byte `json:"salt,omitempty"`
+	Nonce  []byte `json:"nonce,omitempty"`
+}
+
+func (c *APIKeyPair) DecryptCredentialAPISecret() error {
```

Comment:
The field is named "Secret", so it makes sense to call this function "DecryptSecret"
```suggestion
func (c *APIKeyPair) DecryptSecret() error {
```